packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -258,7 +257,9 @@ export default function ProjectNav({
               href={projectNav.alerts as string}
               className={alertIconButtonStyle}
             >
-              {alerts && <div className={alertBadgeStyle}>{alerts}</div>}
+              {typeof alerts === 'number' && (
```

Comment:
Can have the default value of alerts be 0 and then check if alerts > 0 as we don't want to show an icon for 0 alerts.

Reply:
👍